src/screens/ExampleScreen.js

```diff
@@ -1,22 +0,0 @@
-import React, { PureComponent } from 'react';
-import { Text } from 'react-native';
-import { Centered, Page } from '../components/layout';
-import { darkModeThemeColors } from '../styles/colors';
```

Comment:
I deleted this, because this screen was unused. Seems like it was a schema to create new screens off, but it was also outdated